### PR TITLE
taming main runs on ci/cd server

### PR DESCRIPTION
**Pull request by tschm**

### CODE REVIEW

File: experiments/markowitz.py [hunk 1]
```diff
@@ -110,7 +111,7 @@ def markowitz(data: Data, param: Parameters) -> tuple[np.ndarray, float, cp.Prob
     ]
 
     problem = cp.Problem(cp.Maximize(objective), constraints)
-    problem.solve()
+    problem.solve(solver=get_solver())
```
[the remark this answers is not in the document]
Comment (tschm) [reply experiments/markowitz.py:114]: Yes, I don't want to install Mosek on CI/CD but I would prefer to have this function also able to run this during a test or for people without a license

[the remark this answers is not in the document]
Comment (phschiele) [reply experiments/markowitz.py:114]: One of the points we make in the paper is that the solve method works without arguments. I think specifying one here defeats that point a bit (the only reason why we do specify one at all is to ensure replicability). How about this:
I'll update the README and requirements to make the distinction between _exact_ replication (requires mosek license) and _qualitative_ replication (no license required)?

[the remark this answers is not in the document]
Comment (tschm) [reply experiments/markowitz.py:114]: The problem is that the hard dependency on mosek would install it for everybody. Maybe let's add a link to a page where people could download a 30 day license. Not testing code is a sin I could live with

[the remark this answers is not in the document]
Comment (tschm) [reply experiments/markowitz.py:114]: Please assume that the average user has no Mosek license at hand. He will download and run this package and end up with an error message that no license is found. He/she will then argue that this is our fault and whether we can provide the missing license file :-)

**Pull request closed without merging**